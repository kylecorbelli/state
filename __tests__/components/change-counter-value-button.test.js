import 'react-native';
import React from 'react';
import ChangeCounterValueButton from '../../components/change-counter-value-button';
import renderer from 'react-test-renderer';

describe('<ChangeCounterValueButton />', () => {
  it('should match Snapchot', () => {
    const tree = renderer.create(
      <ChangeCounterValueButton
        text='Test Button'
        color='black'
        onPress={() => {}}
      />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
