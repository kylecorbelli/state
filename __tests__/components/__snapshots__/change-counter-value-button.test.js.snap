// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChangeCounterValueButton /> should match Snapchot 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={undefined}
  isTVSelectable={true}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "black",
      "borderBottomLeftRadius": 5,
      "borderBottomRightRadius": 5,
      "borderTopLeftRadius": 5,
      "borderTopRightRadius": 5,
      "height": 70,
      "justifyContent": "center",
      "opacity": 1,
      "width": 130,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#fffddb",
        "fontSize": 50,
      }
    }
  >
    Test Button
  </Text>
</View>
`;
